vendor/licenses/git_submodule/language-bh.dep.yml

```diff
@@ -0,0 +1,41 @@
+---
+name: language-bh
+version: b3dba370c2d669e922fce25cc8319adab82abc3e
+type: git_submodule
+homepage: https://github.com/B-Lang-org/language-bh.git
+license: other
```

Comment:
```suggestion
license: bsd-3-clause
```

Reply:
Good catch!  I had manually added the grammar, before noticing that there was a script.  I diff'd the file generated by the script with my hand-written one, but somehow I missed that they differed in this field.